AppCenterDistribute/AppCenterDistribute/MSDistribute.m

```diff
@@ -1108,7 +1108,7 @@ - (BOOL)openURL:(NSURL *)url {
 }
 
 - (void)applicationWillEnterForeground {
-  if ([self isEnabled] && ![MS_USER_DEFAULTS objectForKey:kMSUpdateTokenRequestIdKey]) {
+  if ([self started] && [self isEnabled] && ![MS_USER_DEFAULTS objectForKey:kMSUpdateTokenRequestIdKey]) {
```

Comment:
There is also `canBeUsed` it' s probably covering all grounds at once. But that's the way to go.